### DISCUSSION TITLE
[Bug] Discussions converted from Issues do not show when default is:open applied

### DISCUSSION THREAD

**Discussion by adamlui** [community/community#54435] [Discussions]
### Bug

When issues are converted into discussions such as  [Use a dedicated DuckDuckGPT conversation #5](https://github.com/kudoai/duckduckgpt/discussions/5), it doesn't appear when user clicks 'Discussions' tab of repo (which auto-applies `is:open` filter):

![image](https://user-images.githubusercontent.com/10906554/236192283-7560ebb2-bbeb-4e52-8a38-5085f881584d.png)

Nothing about the converted discussion indicates it's not open, either:

![image](https://user-images.githubusercontent.com/10906554/236192413-c9416189-2140-4da2-a04b-f3ca78efe782.png)

### Expected behavior

Discussions converted from issues should always appear when filtered by `is:open`

Edit: it finally appears (but a bug is still occuring for it to be delayed)

**Comment by dakshithadissanayaka** [comment]
The discussion in question is related to a bug in GitHub Discussions where converted issues are not shown when the default filter is set to "open". This means that if you convert an issue into a discussion, and then set the filter to only show open discussions, the newly converted discussion will not be visible.

The user who reported the bug suggested that converted discussions should always appear when filtered by "open", regardless of whether they were originally created as discussions or converted from issues. This would ensure that users can easily find all open discussions, regardless of their origin.

The bug has been acknowledged by GitHub, and they are currently investigating a fix. In the meantime, there is no workaround for this issue. Users who want to see converted discussions will need to switch the filter to "all" or manually search for the specific discussion they are looking for.
